build/teamcity/cockroach/nightlies/roachtest_weekly_aws_impl.sh

```diff
@@ -21,4 +21,5 @@ build/teamcity-roachtest-invoke.sh \
   --cluster-id "${TC_BUILD_ID}" \
   --artifacts=/artifacts \
   --artifacts-literal="${LITERAL_ARTIFACTS_DIR:-}" \
+  --parallelism 5 \
```

Comment:
Looks like parallelism defaults to `10`. It makes sense to use consistent values, but I am curious what triggered this change?